ocs_ci/ocs/monitoring.py

```diff
@@ -59,39 +59,20 @@ def validate_pvc_are_mounted_on_monitoring_pods(pod_list):
     """
     for pod in pod_list:
         pod_obj = get_pod_obj(
-            name=pod, namespace='openshift-monitoring'
+            name=pod.name, namespace=pod.namespace
```

Comment:
isn't it defaults.OCS_MONITORING_NAMESPACE

Reply:
done